lib/main.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:math';
+import 'package:dart_collections_journey/names.dart';
+import 'package:word_generator/word_generator.dart';
+
+void main() {
+  runTask1();
+  runTask2();
+  runTask3();
+}
+
+void runTask1() {
+  print('------------------- Task 1 -------------------');
+
+  List<int> numbers = List.generate(100, (index) => Random().nextInt(101));
+
+  print(numbers);
+  print('65-й елемент: ${numbers[64]}');
+
+  numbers.insert(49, 1000000000);
+  numbers.removeWhere((value) => [24, 45, 66, 88].contains(value));
```

Comment:
You might also consider using a tear-off here for conciseness:
https://www.youtube.com/watch?v=OmCaloD7sis
```suggestion
  numbers.removeWhere([24, 45, 66, 88].contains);
```